src/status_im/common/biometric/events.cljs

```diff
@@ -1,120 +1,128 @@
 (ns status-im.common.biometric.events
   (:require
-    [native-module.core :as native-module]
-    [re-frame.core :as re-frame]
     [react-native.biometrics :as biometrics]
-    [react-native.platform :as platform]
+    [schema.core :as schema]
+    [status-im.common.biometric.events-schema :as events-schema]
+    [status-im.common.biometric.utils :as utils]
     [status-im.common.keychain.events :as keychain]
     [status-im.constants :as constants]
     [taoensso.timbre :as log]
     [utils.i18n :as i18n]
     [utils.re-frame :as rf]))
 
-(def android-device-blacklisted?
-  (and platform/android? (= (:brand (native-module/get-device-model-info)) "bannedbrand")))
-
-(defn get-label-by-type
-  [biometric-type]
-  (condp = biometric-type
-    constants/biometrics-type-android (i18n/label :t/biometric-fingerprint)
-    constants/biometrics-type-face-id (i18n/label :t/biometric-faceid)
-    (i18n/label :t/biometric-touchid)))
-
-(defn get-icon-by-type
-  [biometric-type]
-  (condp = biometric-type
-    constants/biometrics-type-face-id :i/face-id
-    :i/touch-id))
-
-(re-frame/reg-fx
- :biometric/get-supported-biometric-type
+(rf/reg-fx
```

Comment:
@clauxx, we are trying to more explicitly separate effects from events.

1. `reg-fx` calls should be made inside `effects.cljs` files.
2. Generally, the names of effects should start with `:effects.*` to differentiate between events.

I believe these are not in the guidelines yet, but we have most of the code following these conventions already.

cc @flexsurfer

Reply:
thanks! was wondering which convention should be used, as there still are quite a few effects without the prefix.